hadoop-ozone/integration-test/src/test/java/org/apache/hadoop/ozone/om/TestObjectStore.java

```diff
@@ -84,17 +84,17 @@ public void testCreateBucketWithBucketLayout() throws Exception {
     store.createVolume(sampleVolumeName);
     OzoneVolume volume = store.getVolume(sampleVolumeName);
 
-    // Case 1: Bucket layout: Empty and OM default bucket layout: OBJECT_STORE
+    // Case 1: Bucket layout: Empty and OM default bucket layout: LEGACY
```

Comment:
By default, the buckets now get created in `LEGACY` bucket layout. Hence, the test modification.